### PR TITLE
#[UIC-797] Enabled Cypress tests on RVF: Dev setup with e2e test cases execution

### PR DESCRIPTION
**Pull request by harshit-kumar-guavus**
Description:

1. Enabled cypress on RVF with sqllab,dashboard and explore.
2. 100 % execution of test cases for all 3 modes - dashboard, explore and sqllab

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -68,42 +68,36 @@ whitelist_externals =
...
     SUPERSET_CONFIG = tests.superset_test_config
     SUPERSET_HOME = {envtmpdir}
 deps =
-    -rrequirements.txt
-    -rrequirements-dev.txt
+    pipenv
 
 [testenv:cypress-explore]
 commands =
-    npm install -g npm@'>=6.5.0'
```
Comment (bipinsoniguavus) [inline tox.ini]: @harshit-kumar-guavus  As discussed, we should ensure that on dev machine, the mentioned `npm` version is available. We can do that with the help of .nvmrc file and choosing the right node version which ships `npm` version greater than `6.5.0`

File: superset/assets/cypress_build.sh [hunk 2]
```diff
@@ -29,6 +29,7 @@ flask run -p 8081 --with-threads --reload --debugger &
 
 #block on the longer running javascript process
 time npm ci
+time npm run install-cypress
```
Comment (bipinsoniguavus) [inline superset/assets/cypress_build.sh]: This step seems to be redundant as the script is installing `cypress` library which is already coming via package.json dependencies.

File: superset/assets/package.json [hunk 3]
```diff
@@ -19,7 +19,8 @@
     "lint-fix": "eslint --fix --ignore-path=.eslintignore --ext .js,.jsx . && tslint -c tslint.json --fix ./{src,spec}/**/*.ts{,x}",
     "sync-backend": "babel-node --preset=@babel/preset-env src/syncBackend.js",
     "cypress": "cypress",
-    "cypress-debug": "cypress open --config watchForFileChanges=true"
+    "cypress-debug": "cypress open --config watchForFileChanges=true",
+    "install-cypress": "npm install cypress@3.1.5"
```
Comment (bipinsoniguavus) [inline superset/assets/package.json]: As mentioned above, `install-cypress` script is redundant. Please remove it.

**Comment by arpit-agarwal** [comment]
We are no longer using cypress

**Pull request closed without merging**